guides/common/modules/con_dhcp-service-providers.adoc

```diff
@@ -0,0 +1,27 @@
+[id="dhcp-serivce-proviers"]
+= DHCP service providers
+
+{Project} can manage IP leases on a DHCP server through a {SmartProxy}.
+This management contains querying for available IP addresses, adding new reservations, and deleting existing reservations from the lease database.
+Note that {SmartProxy} cannot manage subnet declarations.
+
+{SmartProxy} supports the following DHCP providers that you can use to integrate {Project} with your existing DHCP infrastructure or deploy a new one:
+
+`dhcp_isc`:: Managing IP leases on an ISC dhcpd server by using the Object Management Application Programming Interface (OMAPI).
```

Comment:
Should dhcpd have some markup?
```suggestion
`dhcp_isc`:: Managing IP leases on an ISC `dhcpd` server by using the Object Management Application Programming Interface (OMAPI).
```
Might apply elsewhere too.

Reply:
I'm not sure we quote project names like this elsewhere. it should be noted that https://www.isc.org/dhcp/ uses "ISC DHCP"

Reply:
In cases that I refer to the software name, I changed it to "ISC DHCP" - which is the official name.
In cases where it's about the service, I kept dhcpd.